BlueprintUICommonControls/Tests/Sources/AttributedLabelTests.swift

```diff
@@ -23,69 +22,79 @@ class AttributedLabelTests: XCTestCase {
 
         let string = NSAttributedString(string: "Hello, world. This is some long text that runs onto several lines.")
         var element = AttributedLabel(attributedText: string)
-        element.roundingScale = 1
+
+        func constrained() -> Element {
+            element.constrainedTo(width: .atMost(100), height: .atMost(800))
```

Comment:
These tests are now auto-sized instead of using an explicit frame, so they also test that the label measuring is correct.